src/Database/V5/Bloodhound/Types.hs

```diff
@@ -5293,6 +5296,7 @@ data PhraseSuggester =
                   , phraseSuggesterShardSize :: Maybe Int
                   , phraseSuggesterHighlight :: Maybe PhraseSuggesterHighlighter
                   , phraseSuggesterCollate :: Maybe PhraseSuggesterCollate
+                  , phraseSuggesterCandidateGenerators :: [DirectGenerators]
```

Comment:
Wouldn't the addition of this field to an exported type constitute a B version number change instead of a C?